### ISSUE TITLE
Text fails to drop after dragging a little too far to the left of the first column

### ISSUE THREAD

**Issue by NoelCarboni** [microsoft/vscode#103487]
Drag and drop fails to drop if the cursor is too far to the left of the margin, even though the text insertion point cursor is flashing right where it needs to be.

You might say I should move the cursor closer to where I want to drop but a quick way to ensure text is dropped in column 1 is to move the mouse further to the left than it needs to be.

It is beyond frustrating to watch the text insertion point go right to where you want to drop text, then when you let the mouse button up nothing happens.  Then a whole re-selection process has to happen again.

Either the insertion point cursor should be hidden if the drop is not going to be honored, or - ideally - just allow the drop since there's nothing else over there to drop the text on.  Please make sure that if the insertion point cursor is showing, dragged text WILL drop.

Thanks.

**Comment by rebornix** [comment]
![dnd-left](https://user-images.githubusercontent.com/876920/98404552-a76dd480-201f-11eb-8b0e-bfe8810da2e0.gif)

I'm not able to reproduce this issue in latest Insiders, can you help verify if it's what you are seeing as well?

**Comment by NoelCarboni** [comment]
I'm not on the "insiders" track for several reasons.  Is there reason to believe this has been fixed since 1.50.1?

It's possible the drag and drop operation is completing with the cursor outside the bounds of VSCode.  Shouldn't matter - should still drop as the indications on screen have not changed.  Once the cursor is captured the drop operation should complete normally no matter where the mouse button is let up.

**Comment by NoelCarboni** [comment]
Yes, this still needs attention.  It's still broken even with the latest version.

Either:
A.  The drop operation needs to complete normally (recommended).
-or-
B.  The on-screen indication (i.e., insertion cursor) needs to hide when the mouse leaves the application.

**Comment by NoelCarboni** [comment]
@isidorn, can you please facilitate getting this back into view?   This is the one biggest impediment to my productivity using this editor.  Not even one editing session goes by where it doesn't bite me.  When I'm watching the screen during a move, I'm NOT watching the mouse cursor itself, but the dotted vertical line insertion cursor to see where the text will drop.

**Comment by isidorn** [comment]
@NoelCarboni I think you provided enough additional information, let's give some time for @rebornix to reply.
